src/index.html

```diff
@@ -4,8 +4,8 @@
     <title>Firefox Marketplace</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
-
-    <link rel="stylesheet" type="text/css" href="/media/css/bundle.css"/>
+    <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
```

Comment:
I wouldn't do this. It adds an extra request, it'll be something you'll have to reflect in Zamboni, and it'll make local development on spotty connections a pain.